doc/source/reference/general_utility_functions.rst

```diff
@@ -26,6 +28,16 @@ Testing functions
    testing.assert_frame_equal
    testing.assert_series_equal
    testing.assert_index_equal
+   testing.assert_equal
+   testing.assert_almost_equal
+   testing.assert_categorical_equal
```

Comment:
Is it needed to make all those asserts public? 
Eg, what's the pandas-specific value for external projects of using pandas' `assert_equal` or `assert_almost_equal` ?
Also all the EA specific ones could probably be limited to a single general array assert?

Reply:
Thoughts about this? (can also make an issue for it)